test/integration_test.py

```diff
@@ -0,0 +1,89 @@
+import importlib
+
+import pytest
+from colorama import Fore, Style
+from hydra import compose, initialize
+from omegaconf import DictConfig, OmegaConf
+
+system_paths = [
+    "ppo.ff_ippo",
+    "ppo.ff_mappo",
+    "ppo.rec_ippo",
+    "ppo.rec_mappo",
+    "sac.ff_isac",
+    "sac.ff_masac",
+    "q_learning.rec_iql",
+]
+
+discrete_envs = ["gigastep", "lbf", "matrax", "rware", "smax"]
+cnn_envs = ["cleaner", "connector"]
+continuous_envs = ["mabrax"]
+
+
+def _run_system(system_name: str, cfg: DictConfig) -> float:
+    OmegaConf.set_struct(cfg, False)
+    cfg.logger.use_neptune = False  # we never want to log these tests to neptune
+
+    system = importlib.import_module(f"mava.systems.{system_name}")
+    eval_perf = system.run_experiment(cfg)
+
+    print(f"{Fore.CYAN}{Style.BRIGHT}experiment completed{Style.RESET_ALL}")
+    return float(eval_perf)
+
+
+def _get_fast_config(cfg: DictConfig, fast_config: dict) -> DictConfig:
+    # merging fast_config into cfg
+    dconf: dict = OmegaConf.to_container(cfg, resolve=True)
+    dconf["system"] |= fast_config["system"]
+    dconf["arch"] |= fast_config["arch"]
+    cfg = OmegaConf.create(dconf)
+
+    return cfg
+
+
+@pytest.mark.parametrize("system_path", system_paths)
+def test_mava_system(fast_config: dict, system_path: str):
```

Comment:
If ` ppo.ff_ippo` and ` sac.ff_isac `systems will be tested over all the envs (discrete & continuous) we can remove them from here else we'll have 2 extra unnecessary runs I guess 🤔

Reply:
Ye, good point!